README.md

```diff
@@ -99,14 +99,21 @@ You need the following permissions to run this module.
         - `Editor` platform access
         - `Manager` service access
 
+## Note for KMS encryption
```

Comment:
I would move this to the description section, above the Overview title

Reply:
This is under Required IAM access policies section, but can move

Reply:
Oh I see - `Required IAM access policies` usually only covers the access required for the apikey being used, not service to service auth policies. I think it wont be seen here, so suggest move it